aws/amazon-chime-sdk-component-library-react
# MeetingManager throws original error name from getUserMedia call

**Issue #888 :** 
We wrap the original error from `getUserMedia` with the general `Error`. It make it difficult for builders to handle the actual error. With this change, `startAudioInputDevice`, `startVideoInputDevice`, and `setupDeviceLabelTrigger` of `MeetingManager` now throw errors with the original error `name` returned by the `getUserMedia` call.

**Description of changes:**
`startAudioInputDevice`, `startVideoInputDevice`, and `setupDeviceLabelTrigger` of `MeetingManager` now throw errors with the original error `name` returned by the `getUserMedia` call.

For example, before this change the `error` thrown by `startVideoInputDevice` was:
```
{ 
  name: Error, 
  message: MeetingManager failed to select video input device
}
 ```
No it is:
```
{ 
  name: PermissionDeniedError, 
  message: MeetingManager failed to select audio input device. Permission denied by browser
}
```

**Testing**
1. Have you successfully run `npm run build:release` locally?
Yes

2. How did you test these changes?
    1. Modified the roster-test-app to add buttons to call `startAudioInputDevice`, `startVideoInputDevice` manually.
    2. Block the browser device permission.
    3. Using roster test app to join a meeting.
    4. Verify the error log for `setupDeviceLabelTrigger`.
    5. Click button to trigger `startAudioInputDevice`, `startVideoInputDevice` and verify the error log.


7. If you made changes to the component library, have you provided corresponding documentation changes?
No doc needs to be changed.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
